src/components/entity/update.vue

```diff
@@ -155,7 +173,6 @@ const currentVersion = computed(() =>
   table { table-layout: fixed; }
   thead {
     .label-cell { width: 16.66666667%; }
-    .old-value { width: 33.33333333%; }
```

Comment:
By removing this, `.old-value` and `.new-value` get the remaining width not consumed by `.label-cell`. The remaining width will be equally divided between the two because the table has `table-layout: fixed;`.